django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -31,6 +31,9 @@ class ForwardManyToOneDescriptor(Generic[_F]):
     def get_prefetch_queryset(
         self, instances: list[Model], queryset: QuerySet | None = ...
     ) -> tuple[QuerySet, Callable, Callable, bool, str, bool]: ...
+    def get_prefetch_querysets(
+        self, instances: list[Model], querysets: list[QuerySet] | None = ...
+    ) -> tuple[QuerySet, Callable, Callable, bool, str, bool]: ...
```

Comment:
Let's always use generics with type params even if they are just `Any`.

Reply:
Based on my understanding, are you suggesting that, in addition to the code I have added, it is also acceptable to modify the existing code as you have described?

Reply:
I mean when adding new types instead of `QuerySet` and `Callable` use `QuerySet[Any]`, `Callable[..., Any]`, etc